python_modules/libraries/dagster-aws/dagster_aws/pipes/message_readers.py

```diff
@@ -328,13 +350,15 @@ def __init__(
         log_group: Optional[str] = None,
         log_stream: Optional[str] = None,
         log_readers: Optional[Sequence[PipesLogReader]] = None,
+        max_retries: Optional[int] = None,
```

Comment:
I don't think this should be a `None` by default, to me this would signal that no retries are enabled by default. 

Generally `None` is used as a "replacement" for default values for arguments with mutable types, such as `dict` or `list`. This is because using mutable default values is a bad practice. 

Since this argument is an `int` there is no reason to use `None` here, it can just be 

`max_retries: int = 10`